deck/templates/mailing/author_proposal_created.txt

```diff
@@ -0,0 +1,5 @@
+Hi!
+
+This email is a confirmation for the creation of your proposal {{ proposal_title }} to the event {{ event_title }} on SpeakerFight! Remember to share the event so more people can rate your proposal and engaje more participants!
```

Comment:
@berinhard, here we need to know about the event configuration to send the correct message to the proposer. If the event is set to not accept public vote (event.allow_public_voting is set to False) we need to change the last phrase. Maybe only removing the "so more people can rate your proposal" part is enough.